hash.go

```diff
@@ -100,7 +100,7 @@ func (ch *ConsistantHash) Get(key string) (*PeerClient, error) {
 
 // unsafely return the underlying bytes of a string
 // the caller cannot alter the returned byte slice
-func bytes(str string) []byte {
+func strToBytes(str string) []byte {
```

Comment:
Glad you changed this, Probably best to call this `strToBytesUnsafe` or `unsafeSTB` or `toBytesUnsafe` to make sure people know this is unsafe and shouldn't just use it without knowing they can't alter the returned byte slice without causing problems.